running 1 test from ./main.js
Deno.exitCode ... FAILED (0ms)

 ERRORS 

Deno.exitCode => ./main.js:1:6
error: Error: Test case set the exit code to 42.
    at exitSanitizer (ext:cli/40_test.js:110:15)
    at async outerWrapped (ext:cli/40_test.js:129:14)

 FAILURES 

Deno.exitCode => ./main.js:1:6

FAILED | 0 passed | 1 failed ([WILDCARD]ms)

error: Test failed
